### PR TITLE
platform/344: Adding button to apply column selections

### PR DESCRIPTION
**Pull request by robinnong**
- Improvements were made to UX by adding a button to apply selections
- In a previous implementation of the selector, selections were applied when user closes the dropdown. This functionality is now removed.
- See QA instructions in linked ticket

Related to https://github.com/thinkdata-works/namara-table/pull/291 and https://github.com/thinkdata-works/namara-app/pull/5717
Fixes thinkdata-works/platform#344

### CODE REVIEW

**Review by thenduptsering — APPROVED** [review]

**Comment by thenduptsering** [comment]
Would like @broneks to approve this as well before merging

**Comment by robinnong** [comment]
This PR is related to this as well: https://github.com/thinkdata-works/react-multi-select/commit/04e4bc7996dd1deb765523b3bcc5a26c09a9f86f

File: src/dropdown.js [hunk 1]
```diff
@@ -278,6 +285,15 @@ const styles = {
         boxSizing: 'border-box',
         outline: 'none',
     },
+    dropdownFooter: {
+        padding: '0.5em',
```
Comment (broneks) [inline src/dropdown.js:289]: I don't think there are supposed to be quotes here

File: src/dropdown.js [hunk 2]
```diff
@@ -278,6 +285,15 @@ const styles = {
...
+        position: 'sticky',
```
Comment (broneks) [inline src/dropdown.js:290]: does position sticky work when the list is short? Flexbox may be more reliable in this case.

**Pull request merged**